geonames/operations/default.json

```diff
@@ -329,6 +342,19 @@
         ]
       }
     },
+    {
+      "name": "desc_sort_with_after_geonameid",
+      "operation-type": "search",
+      "body": {
+        "query": {
+          "match_all": {}
+        },
+        "sort" : [
+          {"geonameid" : "desc"}
+        ],
+        "search_after": [5000000]
```

Comment:
can you set track total hits to false, we don't need any count here.

Reply:
addressed in b9f095f